util/dataset.go

```diff
@@ -44,6 +44,7 @@ type Stream struct {
 	Input       string     `config:"input" json:"input" validate:"required"`
 	Vars        []Variable `config:"vars" json:"vars,omitempty" yaml:"vars,omitempty"`
 	Dataset     string     `config:"dataset" json:"dataset,omitempty" yaml:"dataset,omitempty"`
+	File        string     `config:"file" json:"file,omitempty" yaml:"file,omitempty"`
```

Comment:
I'm not sure if File is the best name here? Also was undecided if I should add the `.yml` to it or not. Feedback welcome.

Reply:
Does `File` contain the same content as `Template`, but inside the file?

Reply:
Yes, file is basically the path (should we call it path?) and template is the content of the file.

Reply:
Is there is a chance that the same Stream has defined both, `File` and `Template`?

Reply:
The user must define the file path here as File, the Template part will then automatically be read in through the code: https://github.com/elastic/package-registry/pull/335 So yes, both exist.